icon-registry/icons/testing-type-component-solid_x24.svg

```diff
@@ -0,0 +1,11 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none"
```

Comment:
did you mean to leave the width and height in here?

Reply:
Good catch! I did not.